spec/api-net-spec.js

```diff
@@ -1388,8 +1388,7 @@ describe('net module', () => {
         assert.strictEqual(statusMessage, 'OK')
         const headers = response.headers
         assert(typeof headers === 'object')
-        assert.deepStrictEqual(headers[customHeaderName.toLowerCase()],
-          [customHeaderValue])
+        assert.deepStrictEqual(headers[customHeaderName.toLowerCase()], customHeaderValue)
```

Comment:
Does this mean this is a breaking change to a publicly-accessible API?

Reply:
yes, which lends itself to a next question: should this be classified as a fix or a refactor? i can see the argument for both

Reply:
we should also put `BREAKING CHANGE` in the PR description then.

it's not a refactor, because it changes the API.